src/components/profile/profileRoutes.js

```diff
@@ -0,0 +1,16 @@
+import { Route } from 'react-router-dom'
+
+import UserProfile from './UserProfile'
+
+const pages = [
+  {
+    path: ['/users/:id'],
```

Comment:
Instead of this file, could you put the new route in auth routes, just under /users/sign_in etc. ? It doesn't completely belong there, but it'll help us understand the routing. It needs to be below the more specific routes (otherwise /users/sign_in will try to go to a user profile with id "sign_in", and fail).